js/package-lock.json

```diff
@@ -1,6982 +1,8 @@
 {
   "name": "anu_lms",
   "version": "2.0.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
@bjuranic how do you run NPM? why did you revert to lockfileVersion 1?

Reply:
I haven't. I just run npm build && run npm watch.

Reply:
this is now resolved, i have reverted file